packages/glimmer-runtime/lib/compiled/opcodes/dom.ts

```diff
@@ -221,6 +221,34 @@ export class StaticAttrOpcode extends Opcode {
   }
 }
 
+export class ModifierOpcode extends Opcode {
```

Comment:
It makes sense for it to be abstract once we start implementing actions (i.e.: soon). Otherwise we can't create instances and see them in the visualizer.